content/06.methods.md

```diff
@@ -260,8 +260,13 @@ More detail can be found in the [`collapse-rnaseq` analysis module](https://gith
 
 ##### Immune Profiling/Deconvolution
 
-We used the R package immunedeconv [@pmid:31510660; @url:https://github.com/icbi-lab/immunedeconv] to deconvolute, quantify and compare various immune cell types across 21 histologies from the PBTA cohort with xCell [@doi:10.1186/s13059-017-1349-1] and CIBERSORT [@doi:10.1038/nmeth.3337] in the stranded and poly-A collapsed FPKM RNA-seq datasets ([`immune-deconv` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv)).
-Both methods allow between samples (inter-sample), between cell types (intra-sample) and between cancer type (inter-histology) comparisons.
+We used the R package immunedeconv [@pmid:31510660; @url:https://github.com/icbi-lab/immunedeconv] to deconvolute various immune cell types across 21 histologies from the PBTA cohort with `xCell` [@doi:10.1186/s13059-017-1349-1] and `quanTIseq` [@10.1186/s13073-019-0638-6] in the stranded and poly-A collapsed FPKM RNA-seq datasets ([`immune-deconv` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv)). 
```

Comment:
>21 histologies

It looks like this was meant to describe the `broad_histologies`, although I get 22 when I do:
```
> v21 %>%
+   filter(!is.na(broad_histology)) %>%
+   group_by(broad_histology) %>%
+   tally() %>%
+   nrow()
[1] 22
```

Plus, since we are now using select cancer groups for plotting, but actually analyzing all samples, perhaps we just say:

```suggestion
We used the R package immunedeconv [@pmid:31510660; @url:https://github.com/icbi-lab/immunedeconv] to deconvolute various immune cell types across tumors from the PBTA cohort with `xCell` [@doi:10.1186/s13059-017-1349-1] and `quanTIseq` [@10.1186/s13073-019-0638-6] in the stranded and poly-A collapsed FPKM RNA-seq datasets ([`immune-deconv` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv)). 
```

Reply:
> It looks like this was meant to describe the broad_histologies, although I get 22 when I do

Good check, thanks, I hadn't updated this number.